### ISSUE TITLE
abbr attribute of th element described without example.

### ISSUE THREAD

**Issue by programmer3000** [w3c/html#1363]
`abbr` attribute of `<th>` element have no example. It should described with example because it is not typical case and probably not completely clear
https://www.w3.org/TR/html/tabular-data.html#elementdef-th
And text waht does mean "Alternative label to use for the header cell when referencing the cell in other contexts"? What mean "when referencing the cell in other contexts"? That's not mistake? 
Maybe it should be removed of HTML specification as unusable?

**Comment by programmer3000** [comment]
I think `abbr` attribute must be strongly deprecated, because in our day it haven't meaning

**Comment by chaals** [comment]
Why I disagree with deprecating the attribute:

The purpose of the `th/abbr` attribute is to provide a short version of the text in the `th` element.

The common use case in reality is when screen readers announce the relevant headers for a given cell - instead of giving the full text, it may be appropriate to give an abbreviated form.

In principle the same approach could be used when a user agent keeps headers visible while scrolling through a long table. But while this is common in spreadsheets, it seems no browser actually takes advantage of the markup to do so :(

(I certainly agree that because this is not obvious, it should be better explained in the specification).

**Comment by programmer3000** [comment]
@chaals and for what it use (short version of the text)? How it works? What about `<abbr>` element? If they have same functionality sure `abbr` attribute must be deprecated.

**Comment by chaals** [comment]
As noted: In practice today it is used by screen readers. 

For a user who cannot **see** the headings, remembering which ones apply to a given cell is difficult. If you are blind, this is easy to test, but if not, think of a table larger than the screen you have, where the headers are no longer visible.

Screen readers announce the headers that apply to each cell, as the user moves from one to another. This is a extremely helpful behaviour in moderately large tables, and it is a pity that visual user agents don't provide anything like this level of support to their users - especially given that the HTML 4 markup that supports it is 20 years old.

When the "complete" label in a `th` is so long that reading it all each time makes it less help, the `abbr` lets an author give a short version for repeated use - like the screen reader case above.

The functionality is not the same as the `abbr` element. They are both for abbreviations of text, and they have the same name. Once, people hoped browsers would use them to help users in similar ways, but in 20 years only screen readers (and perhaps crawlers) bothered to do anything useful :(

**Comment by programmer3000** [comment]
@chaals omg, it's now so working? How much vodka drink creator of that attribute? Usually cell headers are not too large. So screen reader colud read full cell and it would be right because it would be original name. But now site users must hope for site builder how correctly does it enter the value of `abbr` attribute. For example if table header cell original name "System of a Down members". Nobody knows what value the author will use. Maybe it will "soad m" or "soad members" or or "soadm". And maybe it will too differ from the original and no fact what site builder will add original value of `abbr` attribute "system of a down members" and no fact that assistive user will understand this value

**Comment by prlbr** [comment]
Nobody is forced to use the attribute. Just leave it away if it is not useful in case of your tables. But there are cases where it is helpful.

**Comment by programmer3000** [comment]
@prlbr No no, it's not care case fo assistive technology users.
I think abbr attribute should be remove of HTML specification but browser should voice-render full `<th>` cell. And so then there will not be different interpretation of `<th>header cell name</th>` and `<th abbr="maybe header cell `name"></th>`. And if site builder will make a mistake it will mistake for all users same but not for one user one meaning for assistive user other meaning.

**Comment by edent** [comment]
@programmer3000 please read our [Code of Conduct](https://github.com/w3c/html/blob/master/docs/conduct.md). Specifically:

> Treat each other with respect, professionalism, fairness, and sensitivity to our many differences and strengths, including in situations of high pressure and urgency.

**Comment by programmer3000** [comment]
@edent why close? still not describe `abbr` attribute
